CHANGES.md

```diff
@@ -6,6 +6,47 @@ Version 0.3.0
 
 To be released.
 
+ -  `Dictionary.SetItem` became to have more overloads.  [#7]
+     -  (IKey, string)
+     -  (IKey, byte[])
+     -  (IKey, long)
+     -  (IKey, ulong)
+     -  (IKey, bool)
+     -  (IKey, IEnumerable<IValue>)
+     -  (string, IValue)
+     -  (string, string)
+     -  (string, byte[])
+     -  (string, long)
+     -  (string, ulong)
+     -  (string, bool)
+     -  (string, IEnumerable<IValue>)
+     -  (byte[], IValue)
+     -  (byte[], string)
+     -  (byte[], byte[])
+     -  (byte[], long)
+     -  (byte[], ulong)
+     -  (byte[], bool)
+     -  (byte[], IEnumerable<IValue>)
```

Comment:
Could you format these items as monospace?  Plus, it would be better to mention that these overloads return `Bencodex.Types.Dictionary` instead of `IImmutableDictionary<IKey, IValue>`.